drupal8-apache/7.0/usr/local/share/container/baseimage-30.sh

```diff
@@ -1,41 +1,51 @@
 #!/bin/bash
 
-alias_function do_build do_drupal8_build_inner
-do_build() {
-  do_drupal8_build_inner
-  do_drupal8_install
-}
-
-alias_function do_development_start do_drupal8_development_start_inner
-do_development_start() {
-  do_drupal8_development_start_inner
-  do_drupal8_development_start
-}
+# This is intentionally left as drupal8_functions for the time being.
+# If we need a different set of functions for Drupal 7, then we can load in a
+# different functions file, and keep this main one the same.
+source /usr/local/share/drupal/drupal8_functions.sh
 
-alias_function do_templating do_drupal8_templating_inner
-do_templating() {
-  mkdir -p /app/docroot/sites/default/
-  do_drupal8_templating_inner
+set_drupal_version() {
+  drush core-status drupal-version --format=list | cut -d"." -f 1
 }
 
-do_composer() {
-  # disable original composer in image hierarchy till install ported
-  :
+#####
+# Tasks here happen during the bulding of the Dockerfile, and cannot rely on
+# other services being available.
+#####
+alias_function do_build do_drupal_build_inner
+do_build() {
+  do_drupal_build_inner
+  do_drupal_build
 }
 
-do_drupal8_install() {
-  bash /usr/local/share/drupal8/install.sh
+alias_function do_build do_drupal_development_build_inner
```

Comment:
This is adding to the one declared above on line 17, which might not be what you want. If it isn't then `do_drupal_development_build` in the drupal_functions.sh and the `do_development_build` below can be removed.